ci/kind/test-e2e-kind.sh

```diff
@@ -249,6 +254,10 @@ if $flow_visibility; then
     manifest_args="$manifest_args --feature-gates FlowExporter=true,L7FlowExporter=true --extra-helm-values-file $FLOW_VISIBILITY_HELM_VALUES"
 fi
 
+if $flexible_ipam; then
+    manifest_args="$manifest_args --flexible-ipam --multicast"
```

Comment:
why do we enable multicast in this case?

Reply:
to run these tests:
` --- PASS: TestAntreaIPAMAntreaPolicy/TestGroupNoK8sNP/Case=ACNPAntreaIPAMMulticast (95.79s)
            --- PASS: TestAntreaIPAMAntreaPolicy/TestGroupNoK8sNP/Case=ACNPAntreaIPAMMulticast/Case=MulticastNPIGMPQueryAllow (31.78s)
            --- PASS: TestAntreaIPAMAntreaPolicy/TestGroupNoK8sNP/Case=ACNPAntreaIPAMMulticast/Case=MulticastNPIGMPQueryDrop (34.53s)
            --- PASS: TestAntreaIPAMAntreaPolicy/TestGroupNoK8sNP/Case=ACNPAntreaIPAMMulticast/Case=MulticastNPPolicyEgressAllow (13.30s)
            --- PASS: TestAntreaIPAMAntreaPolicy/TestGroupNoK8sNP/Case=ACNPAntreaIPAMMulticast/Case=MulticastNPPolicyEgressDrop (16.18s)`
            
but ig we don't need to pass it here, I add --multicast flag in workflow.